en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/sample-interceptor-services.md

```diff
@@ -0,0 +1,100 @@
+# Sample Interceptor Services
+
+## Sample 1: JSON to XML transformation for a Legacy Backend - Book Store
+
+Let's say we have a legacy backend named "Book Store". The resource `/books` with the method `POST` accepts an XML
+request to add a new book. If the header `x-user: admin` is not present in the request to the server, it responds
+with the `401` HTTP status code. If the book insertion is successful, it is `200`.
+
+Let's say we require to make the API accepts JSON payload and if the book gets created success,
+respond to it with the HTTP status code `201`, and we are happy about the response returned from the backend so, no need to modify the response.
```

Comment:
```suggestion
respond to it with the HTTP status code `201`. Since the response returned from the backend is safisfactory, there is no need to modify the response.
```